tools/make-nodejs.sh

```diff
@@ -54,6 +54,8 @@ cp platform/nodejs/*.js   $DES/
 cp platform/nodejs/*.json $DES/
 cp LICENSE.txt            $DES/
 
+eslint -c platform/nodejs/eslintrc.json $DES/js $DES/*.js
```

Comment:
The linter is run only on the `make nodejs` command.

As of this patch, there are no errors and a whole bunch of indentation warnings. I'll try to address the indentation warnings in a subsequent patch. They are harmless.

Maybe best to have ESLint installed globally via `npm i eslint -g`.